### ISSUE TITLE
Add Text Styling to Alt Text

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-core#617]
# Issue Description
In https://github.com/cerner/terra-clinical/pull/145 it was noted there is inconsistencies in how browsers displays alt text on image elements. In Chrome and Firefox, the alt text wraps on overflow, while on IE and Safari, the text is cut off. 

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [ ] Bug
- [x] Other

## Expected Behavior
Alt text should be displayed consistently across browsers. 

## Current Behavior
NOTE: The images provided come from the terra-clinical-item-view Accessory example test page. The Accessory elements have a max-width which is causing the text to be wrapped/cut-off.

**Chrome:**
![screen shot 2017-07-14 at 10 38 33 am](https://user-images.githubusercontent.com/14099737/28219188-b3655b1e-6880-11e7-94c3-b833d9e90da2.png)

**Safari:**
<img width="568" alt="screen shot 2017-07-14 at 10 38 49 am" src="https://user-images.githubusercontent.com/14099737/28219196-ba81b8ac-6880-11e7-89b3-5708674a44e4.png">

**Comment by PaulSolDev** [comment]
@bjankord when closing issues please add a brief comment for reference.

**Comment by bjankord** [comment]
Closed as not applicable.
